docs/browser-lifecycle.md

```diff
@@ -0,0 +1,74 @@
+
+---
+
+### User Interaction Phase
+
+**Starts after LCP → Ends when the page begins unloading**  
+Captures user-centric metrics, runtime behavior, and late resource loading.
+
+**All events in this phase may occur multiple times.**
+
+| Event | Description | Semantic Conventions Status | Instrumentation Status                                                                                                                                        |
+|---|---|---|---------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `browser.page_view` | Represents a **soft navigation** (SPA virtual page change). | In review [PR1910](https://github.com/open-telemetry/semantic-conventions/pull/1910) | In review [PR2386](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2386)                                                                      |
+| `browser.web_vital` | Captures Web Vitals metrics such as CLS, INP, and FID that occur after page load. | Merged [WebVitals](https://opentelemetry.io/docs/specs/semconv/browser/events/#webvital-event) | Not created                                                                                                                                                   |
+| `browser.error` | Captures unhandled JavaScript errors. | Not created | Not created                                                                                                                                                   |
+| `browser.console` | Captures browser console messages such as warnings and logs. | Not created | Not created                                                                                                                                                   |
+| `browser.user_interaction` | Captures user input events (clicks, scrolls, keypresses) and their latency. | Not created | Not created                                                                                                                                                   |
+| `browser.performance_resource_timing` | Captures late-loaded assets (ads, analytics, lazy images, etc.). | Not created | Merged (similar) [instrumentation-document-load](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/packages/instrumentation-document-load) |
+
+---
+
+### Unload Phase
+
+**Starts when the user initiates a navigation away, closes the tab, or reloads**  
+Captures the start of the unload process.
+
+| Event | Description | Semantic Conventions Status | Instrumentation Status |
+|---|---|---|---|
+| `browser.page_unload` | Signals the start of the page unload process. Occurs once per page unload. | Not created | Not created |
```

Comment:
I am not sure if this event is necessary. What value would it provide?

Reply:
The unload event is basically deprecated and should not be used due to bfcache. The pagehide event fires also when a document is put into bfcache, which means that the same document can be restored. This implies that it can have a more complex lifecycle than just unload.

Reply:
There is also the visibilitychanged event. This does not indicate the end of a document lifecycle (page view?), but could be useful to understand when the user is actually on the page.